modules/cluster-logging-collector-syslog.adoc

```diff
@@ -0,0 +1,184 @@
+
+To configure the Fluentd *syslog* plug-in, create a configuration file, called `sysconfig.conf`, with the information needed to forward the logs. Then use that file to create a ConfigMap called `syslog` in the `openshift-logging` Namespace, which {product-title} uses when forwarding the logs. On the receiver, configure the *in_syslog* plug-in to receive the logs from {product-title}. For more information on using the in_forward plug-in, see the link:https://docs.fluentd.org/input/syslog[Fluentd documentation].
+
+[IMPORTANT]
+====
+Changes introduced by the new log forward feature modified the support for Fluentd *syslog* plug-in starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap, as described below, to configure the Fluentd *syslog* plug-in. 
+====
+
+You can use multiple syslog servers by specifying separate `<store>` stanzas in the configuration file.
+
+.Sample `sysconfig.conf`
+[source,yaml]
+----
+<store>
+@type syslog_buffered <1>
+remote_syslog rsyslogserver.openshift-logging.svc.cluster.local <2>
+port 514 <3>
+hostname fluentd-4nzfz <4>
+remove_tag_prefix tag <5>
+tag_key ident,systemd.u.SYSLOG_IDENTIFIER <6>
+facility local0 <7>
+severity info <8>
+use_record true <9>
+payload_key message <10>
+</store>
+----
+
+<1> The Fluentd *syslog* plug-in.
+<2> The FQDN or IP address of the syslog server.
+<3> The port number to connect on. Defaults to 514.
+<4> The name of the syslog server.
+<5> Removes the prefix from the tag. Defaults to '' (empty).
+<6> The field to set the syslog key.
+<7> The syslog log facility or source.
+<8> The syslog log severity.
+<9> Determines whether to use the severity and facility from the record if available.
+<10> The key to look on the record to set the payload on the syslog message. Defaults to `message`.
+
+
+// Above definitions from https://github.com/docebo/fluent-plugin-remote-syslog
+
+
+.Sample `syslog` ConfigMap based on the configuration
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: syslog
+  namespace: openshift-logging
+data:
+  sysconfig.conf: |
+    <store>
+     @type syslog_buffered
+     remote_syslog syslogserver.openshift-logging.svc.cluster.local
+     port 514
+     hostname fluentd-4nzfz
+     remove_tag_prefix tag
+     tag_key ident,systemd.u.SYSLOG_IDENTIFIER
+     facility local0
+     severity info
+     use_record true
+     payload_key message
+    </store> 
+----
+
+.Procedure
+
+To configure the Fluentd *syslog* plug-in:
+
+. Create a configuration file named `sysconfig.conf` that contains the following
+parameters within the `<store>` stanza: 
+
+.. Specify the Fluentd *syslog* plug-in type:
++
+----
+@type syslog_buffered <1>
+----
++
+<1> Specify the plug-in to use, either: `syslog` or `syslog_buffered`. 
+
+.. Configure the name, host, and port for your external syslog server:
++
+[source,yaml]
+----
+remote_syslog <fqdn or IP> <1>
```

Comment:
Are we using spaces in our replacement tokens? Or maybe just `<remote>` to encapsulate either.